scripts/test/Muon/plot_widget/plot_widget_presenter_test.py

```diff
@@ -250,6 +250,8 @@ def test_handle_plot_selected_fits_correctly_calls_plot(self):
         fit = FitInformation(mock.MagicMock(), 'GaussOsc',
                              ['MUSR62260; Group; bottom; Asymmetry; MA'],
                              ['MUSR62260; Group; bottom; Asymmetry; MA; Fitted'])
+        self.view.is_plot_diff = mock.Mock
```

Comment:
If we add a `is_plot_diff` method to the [view interface ](https://github.com/mantidproject/mantid/blob/69a834894f26b22df9d14d5ef77024e934dc31c6/scripts/Muon/GUI/Common/plot_widget/plot_widget_view_interface.py) we won't need to create the mock here.

Additionally, we should add a unit test here to check that the correct value is apssed to the model. We can also add a test to the model (plot_widget_model_test.py), to test the `get_fit_workspaces_and_indices` method.